public/assets/ts/admin/editor.ts

```diff
@@ -9,8 +9,7 @@ type EditorNode = {
   path?: (string | number)[];
```

Comment:
I believe this file now needs a `import { showToast } from "./toast";` for all the showToast calls